static/app/views/dashboards/widgetCard/chart.tsx

```diff
@@ -303,6 +303,8 @@ function WidgetCardChart(props: WidgetCardChartProps) {
   };
 
   const chartOptions = {
+    animation: false,
+    notMerge: false,
```

Comment:
maybe link to docs here, or a comment. Its hard to figure out later what each are trying to do

Reply:
Done 👍🏻